app/views/events/_form.html.erb

```diff
@@ -0,0 +1,43 @@
+<%= form_with(model: event, local: true) do |form| %>
+  <% if event.errors.any? %>
```

Comment:
please remove all the logic from your views and put it inside the view helpers `app/helpers/*_helper.rb`

Reply:
Done